website_sale_adj/models/sale_order.py

```diff
@@ -86,21 +88,23 @@ def force_quotation_send(self):
             if email_act and email_act.get("context"):
                 email_ctx = email_act["context"]
                 email_ctx.update(default_email_from=order.company_id.email)
-                # order.with_context(email_ctx).message_post_with_template(
-                #     email_ctx.get('default_template_id'))
                 email_ctx["email_bcc"] = ",".join(
                     [str(partner.email) for partner in order.message_partner_ids]
                 )
                 email_ctx["reply_to"] = order.message_get_reply_to(
                     [order.id], default=email_ctx["default_email_from"]
                 )[order.id]
-                self.env["mail.template"].browse(
-                    email_ctx.get("default_template_id")
-                ).with_context(email_ctx).send_mail(order.id)
+                order.with_delay().send_confirmation_email(email_ctx)
```

Comment:
@TimLai125 Can we add a description to the job, please?

Reply:
You mean passing an argument to the job to set with meaning name or a comment for this line?

Reply:
@TimLai125 The former.